### DISCUSSION TITLE
About the showcase page of https://astro.build

### DISCUSSION THREAD

**Discussion by tiwentichat** [withastro/roadmap#784] [Proposal]
### Body

# Summary

Would you like to kindly open source the repo of Showcase page of the official website?

# Background & Motivation

The showcase page is really cool and nice, I like it so much, wish for the repo be open sourced too.

# Goals

compared those feathers page in some websites, this page is more elegant and clear.

1 picture is better than 1000 words

**Comment by florian-lefebvre** [comment]
It is! https://github.com/withastro/astro.build/blob/main/src/pages/showcase/[...page].astro

**Reply by tiwentichat** [reply]
Get it! Merci! Florian : )
